### PR TITLE
feature(core): Create voting script for GASETH-TWAP-1Mx1M price identifier

### PR DESCRIPTION
**Pull request by smb2796**
**Motivation**

This script will allow UMA voters to query for median 30 day gas prices through Google Big Query.

**Summary**

To run this price feed locally, you need to set a GOOGLE_APPLICATIONS_CREDENTIALS environment variable with GCP admin or BigQuery permissions. 

The BQ query was included in a separate file for readability.

The following packages are also added to core/package.json.
- [highland](https://www.npmjs.com/package/highland)
- [google-cloud/bigquery](https://www.npmjs.com/package/@google-cloud/bigquery)

At a later date, this script should be generalized to accept any GASETH price identifier or any other price identifier that uses Google BQ.

### CODE REVIEW

File: packages/core/scripts/local/getGasEthHistoricalPrice.js
Comment (smb2796) [inline packages/core/scripts/local/getGasEthHistoricalPrice.js:75]: `toFixed` introduces a slight rounding error when scaling the price identifier. As an example, a return value of `.086` is changed to `0.085999999999999993`. Any idea why this is?

File: packages/core/scripts/local/GasEthQuery.js
Comment (chrismaree) [inline packages/core/scripts/local/GasEthQuery.js:17]: this is the same bigquery we had back when you first designed the uGAS feeds, right? before moving over to the twap pattern?

File: packages/core/scripts/local/getGasEthHistoricalPrice.js
Comment (chrismaree) [reply packages/core/scripts/local/getGasEthHistoricalPrice.js:75]: I suspect this is because your `precisionToUse` is set to 18. normally this value is closer to 5. you might need to do the `toFixed` logic differently seeing that javascript does not work well with this kind of tiny number.

One option would be to use the native bignumber `.toFixed` which will properly work. ([docs](https://mikemcl.github.io/bignumber.js/#toFix)). To do this try out something like:

```
toBN(fromWei(queryPrice.toString())).toFixed(precisionToUse)
```

where you convert to a BN rather than to a `Number`.

File: packages/core/scripts/local/getGasEthHistoricalPrice.js
Comment (chrismaree) [inline packages/core/scripts/local/getGasEthHistoricalPrice.js:7]: nit: should the example contain `identifier`?

**Review by chrismaree — APPROVED** [review]
This looks good to me! I had a nit and a few questions but overall I'm happy :)

File: packages/core/scripts/local/GasEthQuery.js
Comment (smb2796) [reply packages/core/scripts/local/GasEthQuery.js:17]: Yes it is the same.

File: packages/core/scripts/local/getGasEthHistoricalPrice.js
Comment (smb2796) [reply packages/core/scripts/local/getGasEthHistoricalPrice.js:7]: Hm yeah probably. Was initially thinking this was only usable for one identifier, but it would also work for the `uGAS-FEB21` and `uGAS-MAR21` identifiers, so I will include.

File: packages/core/scripts/local/getGasEthHistoricalPrice.js
Comment (smb2796) [reply packages/core/scripts/local/getGasEthHistoricalPrice.js]: Cool, thank you. I addressed with

`BigNumber(fromWei(queryPrice.toString())).toFixed(precisionToUse)`

**Review by nicholaspai — APPROVED** [review]
LGTM

File: packages/core/scripts/local/getGasEthHistoricalPrice.js
Comment (pemulis) [inline packages/core/scripts/local/getGasEthHistoricalPrice.js:44]: Is there ever a situation where someone would want to query with the current time and not a specified timestamp?

File: packages/core/scripts/local/getGasEthHistoricalPrice.js
Comment (pemulis) [inline packages/core/scripts/local/getGasEthHistoricalPrice.js:57]: `logging`

File: packages/core/scripts/local/getGasEthHistoricalPrice.js
Comment (nicholaspai) [reply packages/core/scripts/local/getGasEthHistoricalPrice.js:44]: Yeah for sanity checking or a quick run for example

**Comment by nicholaspai** [comment]
@smb2796 let's merge this after you merge in master!

File: packages/core/package.json
Comment (smb2796) [inline packages/core/package.json:37]:
```suggestion
    "bignumber.js": "^8.0.1",
```

**Pull request merged**
